include/wally_map.h

```diff
@@ -462,6 +462,34 @@ WALLY_CORE_API int wally_map_keypath_add(
     const uint32_t *child_path,
     size_t child_path_len);
 
+/**
+ * Convert and add a public key, leaves and path to taproot leafhash/keypath maps.
+ *
+ * :param hashes_map_in: The taproot_leaf_hashes map to add to.
+ * :param paths_map_in: The taproot_leaf_paths map to add to.
+ * :param pub_key: The public key or extended public key to add.
+ * :param pub_key_len: Length of ``pub_key`` in bytes. Must be `BIP32_SERIALIZED_LEN`
+ *|    for an extended bip32 public key, or `EC_XONLY_PUBLIC_KEY_LEN`
+ *|    for a public key.
```

Comment:
I think only EC_XONLY_PUBLIC_KEY_LEN is valid?

Reply:
done